tck/src/main/java/org/eclipse/microprofile/openapi/tck/ModelReaderAppTest.java

```diff
@@ -153,15 +153,19 @@ public void testAPIResponse(String type) {
         vr.body("paths.'/availability'.get.responses.'404'.description", equalTo("No available flights found"));
     }
 
-    private void testAvailabilityGetParamater(ValidatableResponse vr) {
+    @RunAsClient
+    @Test(dataProvider = "formatProvider")
+    public void testAvailabilityGetParameter(String type) {
+        ValidatableResponse vr = callEndpoint(type);
         String availabilityParameters = "paths.'/availability'.get.parameters";
```

Comment:
This endpoint should be changed to /availabilityModel (to match the change to the app).

The size check (below) will fail when you make that change because departureDate is a reference and wouldn't have a 'name'. So adjust the size check as well as remove departureDate from the 'hasItems' check.

Reply:
Actually the size check wouldn't fail (it would still be 6 parameters). So only the hasItems check would have to be altered.

Reply:
The size of the collection will still be 6, but the check for the name will fail with departureDate there, so I'll remove that.